Zend/zend_operators.c

```diff
@@ -726,7 +738,7 @@ ZEND_API void multi_convert_to_string_ex(int argc, ...) /* {{{ */
 }
 /* }}} */
 
-ZEND_API zend_long ZEND_FASTCALL _zval_get_long_func(zval *op) /* {{{ */
+static zend_long ZEND_FASTCALL _zval_get_long_func_ex(zval *op, zend_bool silent) /* {{{ */
```

Comment:
Might be worth making this into a `zend_always_inline` function and then have `_zval_get_long_func` and `_zval_get_long_func_nonsilent` or something. Dunno how compiler deals with this atm, haven't benchmarked.